LayoutTests/http/tests/resourceLoadStatistics/add-blocking-to-redirect.html

```diff
@@ -11,7 +11,7 @@
     jsTestIsAsync = true;
 
     const partitionHost = "127.0.0.1:8000";
-    const thirdPartyOrigin = "http://localhost:8000";
+    const thirdPartyOrigin = "https://localhost:8443";
```

Comment:
Should we explicitly set samesite=none on the cookies (e.g., `firstPartyCookieName + "&value=value;samesite=none"` below?

Reply:
SameSite=None is explicitly set in set-cookie.py